.github/workflows/_move_tests.yml

```diff
@@ -26,4 +26,24 @@ jobs:
           tool: nextest
       - name: Run move tests
         run: |
-          cargo nextest run -p iota-framework-tests -- unit_tests::
+          cargo nextest run -E
+          '(package(iota-framework-tests) and test(tests::))
```

Comment:
To what is `tests::` referring ?

Reply:
That is the naming scheme of the tests in that package. There are also `build` tests. I'm actually not sure whether we should just do all of the tests or not.

Reply:
I changed this so that it runs all the tests in framework tests